06_gpu/stable_diffusion_cli.py

```diff
@@ -107,7 +116,7 @@ def __enter__(self):
         self.pipe = diffusers.StableDiffusionPipeline.from_pretrained(cache_path, scheduler=euler).to("cuda")
         self.pipe.enable_xformers_memory_efficient_attention()
 
-    @stub.function(gpu=modal.gpu.A100())
+    @stub.function(gpu=modal.gpu.A100(), secret=modal.Secret({"SAFETENSORS_FAST_GPU": "1"}))
```

Comment:
`SAFETENSORS_FAST_GPU` needs to be enabled when we load the models in `__enter__`. When adding it here, would it be available there?

Reply:
yeah it will be available. this is a confusing thing about how the lifecycle methods work – and why i want to move the decorator to be on the class instead